bgpd/bgp_community.c

```diff
@@ -451,9 +451,10 @@ static void set_community_string(struct community *com, bool make_json)
 			val = comval & 0xFFFF;
 			char buf[32];
 			snprintf(buf, sizeof(buf), "%u:%d", as, val);
-			strlcat(str, bgp_community2alias(buf), len);
+			const char *com2alias = bgp_community2alias(buf);
+			strlcat(str, com2alias, len);
```

Comment:
add a blank line here.  I'll get this in after that

Reply:
Fixed.